playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_positioning.html.erb

```diff
@@ -0,0 +1,67 @@
+            <%= pb_rails("caption", props: { text: "value: 200", size: 'md' }) %>
+        <% end %>
+
+            <%= pb_rails("card", props: { classname: "three-zIndex one-z-index-example", z_index: "3", shadow: "deeper" }) do %>
+               <%= pb_rails("body", props: { text: "$z_3" }) %>
+            <%= pb_rails("caption", props: { text: "value: 300", size: 'md' }) %>
+        <% end %>
+
+            <%= pb_rails("card", props: { classname: "four-zIndex one-z-index-example", z_index: "4", shadow: "deeper" }) do %>
+            <%= pb_rails("body", props: { text: "$z_4" }) %>
+            <%= pb_rails("caption", props: { text: "value: 400", size: 'md' }) %>
+        <% end %>
+
+            <%= pb_rails("card", props: { classname: "five-zIndex one-z-index-example", z_index: "5", shadow: "deeper" }) do %>
+             <%= pb_rails("body", props: { text: "$z_5" }) %>
+            <%= pb_rails("caption", props: { text: "value: 500", size: 'md' }) %>
+        <% end %>
+
+            <%= pb_rails("card", props: { classname: "six-zIndex one-z-index-example", z_index: "6", shadow: "deeper" }) do %>
+             <%= pb_rails("body", props: { text: "$z_6" }) %>
+            <%= pb_rails("caption", props: { text: "value: 600", size: 'md' }) %>
+        <% end %>
+
+            <%= pb_rails("card", props: { classname: "seven-zIndex one-z-index-example", z_index: "7", shadow: "deeper" }) do %>
+             <%= pb_rails("body", props: { text: "$z_7" }) %>
+            <%= pb_rails("caption", props: { text: "value: 700", size: 'md' }) %>
+        <% end %>
+
+                <%= pb_rails("card", props: { classname: "eight-zIndex one-z-index-example", z_index: "8", shadow: "deeper" }) do %>
+             <%= pb_rails("body", props: { text: "$z_8" }) %>
+            <%= pb_rails("caption", props: { text: "value: 800", size: 'md' }) %>
+        <% end %>
+
+                <%= pb_rails("card", props: { classname: "nine-zIndex one-z-index-example", z_index: "9", shadow: "deeper" }) do %>
+             <%= pb_rails("body", props: { text: "$z_9" }) %>
+            <%= pb_rails("caption", props: { text: "value: 900", size: 'md' }) %>
+        <% end %>
+
+                <%= pb_rails("card", props: { classname: "ten-zIndex one-z-index-example", z_index: "10", shadow: "deeper" }) do %>
+             <%= pb_rails("body", props: { text: "$z_10" }) %>
+            <%= pb_rails("caption", props: { text: "value: 1000", size: 'md' }) %>
+        <% end %>
+    </div>
+    <br>    
+<% end %>
+     <% react_contents = Rails.root.join("app/views/pages/code_snippets/position_token.txt").read %>
+
+        <%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper", shadow: "deeper" }) do %>
+            <div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+              <pre class="highlight pb_none"><%= raw rouge(react_contents, "react") %></pre>
+            </div>        
+        <% end %>
```

Comment:
Take a look at another section for the fresh way to do this. You might need to copy an example to yours to nail the settings